src/views/docs/en/reference/cli/deploy.md

```diff
@@ -18,7 +18,7 @@ arc deploy [production|static|direct]
 ## Flags
 
 - `[dirty, --dirty, -d]` Overwrite staging Lambda with local source. A faster way to deploy and test small changes to individual functions without redeploying an entire stack. 
```

Comment:
I think line 15 should also read `dirty` instead of `direct`.

Reply:
Thanks for bringing this up. I think these docs and instructions need a review in terms of `direct`/`dirty` deploy support. I, for example, deploy directly without any flag at all and just pass the path to the function I want to deploy directly, i.e. `arc deploy src/http/get-index` - but that all can be reviewed and updated independently of this pull request I think.